packages/ferric/src/template.ts

```diff
@@ -0,0 +1,58 @@
+export function cargoTomlTemplate(projectName: string) {
```

Comment:
I wonder whether the CLI should rather take the up-to-date standard Cargo library template and modify it, rather than having a template that may go out of date over time. There are pros and cons either way, of course.

Reply:
Unlikely that will keep changing. Better to have it predefined. Only thing that can most likely change are the edition year.